about.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About</title>
+    <link href="style.css" rel="stylesheet">
+  </head>
+  <body>
+    <header>
+        <a class="log"><img src="img/quality logo.jpeg" alt="logo"></a> 
+    
+        <nav>
+          <ul class="header-list">
+            <li><a href="index.html" class="visited">Home</a></li>
```

Comment:
- [ ] In the mobile version the `Home` link doesn't work as expected, please 🙏🏽 fix it.